content/docs/hooks-overview.md

```diff
@@ -56,25 +56,25 @@ function ExampleWithManyStates() {
 
-Hooks are functions that let you “hook into” React state and lifecycle features from function components. Hooks don't work inside classes -- they let you use React without classes. (We [don't recommend](/docs/hooks-intro.html#gradual-adoption-strategy) rewriting your existing components overnight but you can start using Hooks in the new ones if you'd like.)
+'Hooks' ऐसे फंक्शन हैं जो आपको फ़ंक्शन कॉम्पोनेंट्स से React state "में Hooks" और लाइफ साइकिल की सुविधा देते हैं। 'Hooks' क्लासेस के अंदर काम नहीं करते - वे आपको क्लासेस के बिना React का उपयोग करने देते हैं। (हम [सलाह नहीं देते](/docs/hooks-intro.html#gradual-adoption-strategy) की आप अपने मौजूदा कॉम्पोनेंट्स को रात भर में Hooks में परिवर्तित करदे लेकिन यदि आप चाहें तो नए कॉम्पोनेंट्स में Hooks का उपयोग शुरू कर सकते हैं।)
 
-React provides a few built-in Hooks like `useState`. You can also create your own Hooks to reuse stateful behavior between different components. We'll look at the built-in Hooks first.
+React `useState` जैसे कुछ बने बनाये Hooks प्रदान करता है। आप विभिन्न कॉम्पोनेंट्स के बीच stateful बिहेवियर का पुन: उपयोग करने के लिए अपने स्वयं के Hooks भी बना सकते हैं। हम बने बनाये हुक को पहले देखते है।
 
->Detailed Explanation
+>विस्तृत विवरण
 >
->You can learn more about the State Hook on a dedicated page: [Using the State Hook](/docs/hooks-state.html).
+>आप समर्पित पृष्ठ पर state hook के बारे में अधिक जान सकते हैं। [State Hook का उपयोग करना](/docs/hooks-state.html)
 
 ## ⚡️ Effect Hook {#effect-hook}
 
-You've likely performed data fetching, subscriptions, or manually changing the DOM from React components before. We call these operations "side effects" (or "effects" for short) because they can affect other components and can't be done during rendering.
+आपने संभवतः पहले डेटा फेचिंग, सब्स्क्रिप्शन, या मैन्युअल रूप से DOM को React कॉम्पोनेंट्स से बदलकर दिखाया हुआ है। हम इन ऑपरेशनों को "साइड इफेक्ट्स" (या शॉर्ट के लिए "effects") कहते हैं क्योंकि वे अन्य कॉम्पोनेंट्स को प्रभावित कर सकते हैं और रेंडरिंग के दौरान नहीं किये जा सकते।
 
-The Effect Hook, `useEffect`, adds the ability to perform side effects from a function component. It serves the same purpose as `componentDidMount`, `componentDidUpdate`, and `componentWillUnmount` in React classes, but unified into a single API. (We'll show examples comparing `useEffect` to these methods in [Using the Effect Hook](/docs/hooks-effect.html).)
+Effect Hook, `useEffect`, एक फ़ंक्शन कॉम्पोनेंट्स से साइड इफेक्ट करने की क्षमता देता है। यह उसी प्रकार में कार्य करता है जो `componentDidMount`,` componentDidUpdate`, और `componentWillUnmount`` React classes में करते है , लेकिन यह एक ही API में एकीकृत है। (हम इन तरीकों की तुलना में `useEffect` की तुलना करने वाले उदाहरण यहाँ दिखाएंगे [State Hook का उपयोग करना](/docs/hooks-effect.html)।)
 
-For example, this component sets the document title after React updates the DOM:
+उदाहरण के लिए, DOM को React के अपडेट के बाद यह कॉम्पोनेंट डॉक्यूमेंट का शीर्षक सेट करता है: 
```

Comment:
`DOM को React के अपडेट के बाद यह कॉम्पोनेंट डॉक्यूमेंट का शीर्षक सेट करता है: ` => `React के DOM को अपडेट करने के बाद यह कौम्पोनॅन्ट डॉक्यूमेंट का शीर्षक सेट करता है:`

Reply:
I still feel this is not correct. what about this:
`उदाहरण के लिए, React DOM को अपडेट करने के बाद कौम्पोनॅन्ट डॉक्यूमेंट का शीर्षक सेट करता है:`